_docs/technical/2017-05-14-hd-wallets.md

```diff
@@ -9,15 +9,13 @@ visible: true
 
 # HD wallets
 
-<!-- @martoon TODO: perhaps this part better fits to protocols section -->
-
 HD wallets is a feature which allows users to derive keys in deterministic way
 from a common seed. Basically, you generate an initial secret key `SK₀` out of a
 random seed. Then you can derive children `SK₀-₀`, `SK₀-₁` out of `SK₀`. From
 these children, you can derive `SK₀-₀-₀`, `SK₀-₀-₁`, `SK₀-₁-₀` and so on
 (derivations for a tree of arbitrary depth).
 
-<!-- Advertisement: for subscripts symbols: https://en.wikipedia.org/wiki/Unicode_subscripts_and_superscripts -->
+<!-- For subscripts symbols: https://en.wikipedia.org/wiki/Unicode_subscripts_and_superscripts -->
```

Comment:
If you are on Linux, you can configure XCompose to type in sub- and superscripts directly. Check out this: https://help.ubuntu.com/community/ComposeKey